libs/shared/ui/nuverial/src/lib/components/confirmation-modal/confirmation-modal.component.scss

```
@@ -37,6 +37,9 @@
 
     &__actions {
       padding: 32px 24px 16px;
+      nuverial-button:first-child {
+        margin-right: rem.fromPx(12px);
+      }
```

`__actions` is flex, all this can be:

```suggestion
gap: rem.fromPx(12px)
```